packages/wonder-blocks-labeled-field/src/theme/default.ts

```diff
@@ -6,9 +6,8 @@ const theme = {
             paddingBlockEnd: {
                 labelWithDescription: sizing.size_040,
                 labelWithNoDescription: sizing.size_120,
-                description: sizing.size_120,
-                helperTextSectionWithContent: sizing.size_120,
             },
+            spacingBetweenHelperText: sizing.size_120,
```

Comment:
Simplifying component tokens too. `description` and `helperTextSectionWithContent` have been combined into `spacingBetweenHelperText`